### DISCUSSION TITLE
Anyone noticed Voice Control glitches on Labs?

### DISCUSSION THREAD

**Discussion by sjmarcy2** [gopro/labs#1172] [Q&A] [answered]
With Voice Control ON in the Dashboard from startup, it shows ON status if I check, yet does not work.   But if I toggle it OFF then back ON, now it works and will stay on for the session.    

Or from the main screen, just swipe left or right to change camera modes, and then it is back on.   Maybe there is a setting I have to revise?    Labs features used at the moment are BITR and GCMP.   Thanks.

**Comment by dnewman-gpsw** [comment, marked as answer]
It was recently reported on HERO13 (only) and there is a fix coming. Currently the only workaround is the menu.

**Comment by sjmarcy2** [comment]
Labs FW v1.30.70 still has Voice  Control woes not present on V1.30.

Plus gained the new extra beeps from v1.30, even if beeps (now called "Camera Volume" in the Audio tab) are turned off.

**Reply by dnewman-gpsw** [reply]
The voice control is partly fixed as you can now enable and disable again through QR Codes (that was also broken.) I agree that is still doesn't seem to be honoring the menu state at boot up. Just like the menu, it is seem to want to be toggled. A workaround: scan *BOOT="oVC0oVC", which will force voice control on a every boot.

I haven't noticed any beeps, my volume is always off.  When do they occur?  If they are in stock 1.30, that might be a bug I can report.

**Reply by sjmarcy2** [reply]
the beep glitch...it's from stock 1.30.    found out the cause.   IF you toggle dashboard in a given preset's settings...AND it wants to change your Dashboard overall settings you already made...THEN odd stuff can happen.   Just look for a dimmed Dashboard screen....if you don't see that it will not act strange.   Plus they put beeps into an Audio / Camera Volume re-naming change, which can delay sorting this out when doing a shoot,  Cuz you could be looking for "Beep Volume" like on some prior models.   Basic little-thing sorting really.    On the Voice Control...fixed on 1.30, still funky on 1.30.70 Labs

**Reply by FlyTieGuy** [reply]
Hi. Any update on a fix for the Voice Control glitch? Just for info I use the Labs Hindsight  function and the suggested boot command above seems to override that. Is there a workaround to allow both to be in place. Thanks

**Reply by dnewman-gpsw** [reply]
The fix was either in the December release or hopefully in the one upcoming.

**Reply by casperskulls** [reply]
I just bought my hero 13 black maybe 3 weeks ago and the voice commands don't work

**Reply by dnewman-gpsw** [reply]
More details, in what way?  What Labs version?

**Reply by casperskulls** [reply]
The hero 13 black is brand new so I'd imagine the latest version of labs, not sure what you mean by labs version, but in more detail voice commands don't work meaning I can say gopro turn off and nothing happens, I can give any command and nothing happens.. I've turned voice commands on and off and nothing changes. Just doesn't work..

**Reply by dnewman-gpsw** [reply]
This is the forum for custom GoPro Labs firmware, it does not come with the camera, so you are post in the wrong place. 

To learn about GoPro Labs: https://gopro.com/en/us/info/gopro-labs
To ask user question on stock GoPro feature, maybe reddit: https://www.reddit.com/r/gopro/

**Reply by FlyTieGuy** [reply]
Hi. I have now updated with both the new GoPro firmware v2.02 and Labs firmware 2.02.70 and the voice control issue still stands. I’ve also tried the *BOOT="oVC0oVC", suggestion and it doesn’t help. Any thoughts?

**Reply by dnewman-gpsw** [reply]
Please exactly state what you see is the issue, no has recently, as Voice control is working in my testing.

**Reply by FlyTieGuy** [reply]
I’m using labs to turn on hindsight on boot up and to keep hindsight running basically 24/7.  With this setup on Hero 12 I used GoPro start recording and GoPro stop recording to operate recording. This worked until I recently transferred everything to Hero 13 and it doesn’t work. If I turn hindsight off then voice control still does not work. If I toggle the voice control button off and on then voice control works with hindsight both off and on

**Reply by dnewman-gpsw** [reply]
So this is only with Hindsight?  Voice control just works without (if you started without Hindsight?) You said you can an issue with 2.02 and 2.02.70. If that is the case, this is not a Labs issue.

**Reply by FlyTieGuy** [reply]
In order to try this without using the lab’s commands how do I remove all lab firmware and boot commands

**Reply by dnewman-gpsw** [reply]
*BOOT=""  Clears the boot commands directly
!RESET  Erases all the permanent settings (including BOOT commands.)

**Reply by FlyTieGuy** [reply]
Thanks. Does that also remove the labs firmware

**Reply by dnewman-gpsw** [reply]
No, only as firmware update can replace firmware.

**Reply by FlyTieGuy** [reply]
I reset the camera to factory settings. When I switched on the voice command worked. When I applied the GoPro labs commands it stopped working until I toggled the voice command off and on

**Reply by dnewman-gpsw** [reply]
Which Labs commands? Add them one at a time.

**Reply by FlyTieGuy** [reply]
![image](https://github.com/user-attachments/assets/2be085b8-c579-4de3-9ede-38f444369bdd)

**Reply by FlyTieGuy** [reply]
![image](https://github.com/user-attachments/assets/124de75b-1a8b-4f67-b75b-af00b601da6d)

**Reply by FlyTieGuy** [reply]
![image](https://github.com/user-attachments/assets/31cefd33-6a59-42a7-bc05-053b7603321e)

**Reply by FlyTieGuy** [reply]
In that order

**Reply by dnewman-gpsw** [reply]
which one broke it? I mean you should test it one at a time.

**Reply by dnewman-gpsw** [reply]
I am able to reproduce the error.

**Reply by FlyTieGuy** [reply]
Hi any progress on this one.

**Reply by michaelmcdonald** [reply]
@dnewman-gpsw: any progress on this by chance?

**Reply by dnewman-gpsw** [reply]
try https://github.com/gopro/labs/discussions/1428

**Reply by michaelmcdonald** [reply]
I'll try it out and report back! Thanks for linking it!

**Comment by DemonViewLLC** [comment]
I've noticed it when using the Max Lens Mod.  The voice commands don't work on startup (using normal power up, not quik capture).  You have to turn off and on the voice commands in the menu to make them work again.

Using the standard camera lenses, no such issue.  Voice commands work as normal when the camera is turned on.

Haven't seen if the issue is present when using other lens mods (Macro, ND filters etc).

**Comment by FlyTieGuy** [comment]
Ok. Thanks for your help so far. Getting late here so I’ll reply tomorrow

**Comment by FlyTieGuy** [comment]
Good man. Do you need anything further from me in the meantime?

**Comment by LordWimmi** [comment]
I have the same issue with GP13 since firmware 1.30 and now still with 2.02 (only Labs). Very annoying!!! My workaround:
**_*BOOT="!Lbt"!SAVEbt=v0v1_**
(for safety reasons you better put the script on the micro SD-Card)
It works for me! But that has 2 disadvantages:
1. Voice Control is alway on when you start the GP, even if turned off before
2. If you use the QuickCaptureButton, you will probably experience short sound issues at the point where the script is loaded while GP is already recording.

So, I hope that bug will be finally fixed soon by a working firmware! Labs firmware is indispensable for me :(

**Reply by FlyTieGuy** [reply]
When I add this script it seems to override another boot command I have in place. Can I add multiple boot commands?

**Reply by dnewman-gpsw** [reply]
You can combine for one boot command, as there can be only one.  Or just install this https://github.com/gopro/labs/discussions/1428 and report whether the voice command issue goes away.

**Comment by sjmarcy2** [comment]
For now I just have the Hero 13 default to Time Lapse / Warp mode.   

So after powering on, one routine-memorized side button press gets me to my true default, video.    Not bad once done a few times.   That way I don't have to cycle thru the options with more button presses. 

It will all be sorted soon in any event.

**Reply by motigchelaar** [reply]
What makes you think it will be sorted soon?  This has been an issue for months already...

**Reply by sjmarcy2** [reply]
Because a few posts above, Mr Newman noted (2 weeks ago) that he can reproduce the issue?

So, perhaps the next Labs update..

**Comment by motigchelaar** [comment]
I have this same issue on my GP13 with latest labs firmware. No matter what lens or ND-filter is attached. I have to either manually toggle voice control off and back on or switch to a different camera mode - and then back again.

I may have to resort to using the boot script method.  Annoying nonetheless.

**Comment by LordWimmi** [comment]
I just tested version https://bit.ly/LABS_HERO13_Aug8_2_02_72 I restarted the GP for 20 times for testing voice control. The bug seems to have been fixed. :))) Finally! I'm very happy about that. I hope it stays that way.

**Reply by michaelmcdonald** [reply]
Can also add to this positive news. Have been running the linked firmware above since the 9th and have repeatedly used voice control with zero issues (this is through multiple reboots of my GoPro as well as recording multiple videos). 

This might be the winner!
